UI/properties-view.cpp

```diff
@@ -1910,16 +1910,9 @@ void WidgetInfo::GroupChanged(const char *setting)
 						  : true);
 }
 
-void WidgetInfo::EditListReordered(const QModelIndex &parent, int start,
-				   int end, const QModelIndex &destination,
-				   int row)
+void WidgetInfo::EditListReordered(const QModelIndex &, int, int,
+				   const QModelIndex &, int)
```

Comment:
Arguments can be removed

Reply:
Needs to be a auto generated slot.

Reply:
See https://github.com/obsproject/obs-studio/pull/8167#discussion_r1118067680

Reply:
Fixed